### PR TITLE
Tnw-clone  homepage

### PR DESCRIPTION
**Pull request by chrystalme**
Please, take a look at my project so far.

### CODE REVIEW

**Review by tirthajyoti-ghosh — CHANGES_REQUESTED** [review]
# ⚠️ Invalid

The PR description and the README is not professional. Please always make sure that your README is complete and add a description of the project in your PR description.

I am marking this project invalid as it's considered an unprofessional project submission. Please request a review when you have made it professional.

**Review by jubaan — CHANGES_REQUESTED** [review]
# :rotating_light: :rotating_light: :rotating_light: :construction: Status: Invalid :construction: :rotating_light: :rotating_light: :rotating_light:

The project is marked as invalid for the next reasons.

- [ ] PR should be descriptive.

- [ ] Project should have a professional README.

- [ ] Project should be a working finished version.

- [ ] All linter checks should Pass.

DON'T submit your project until the project is finished. If you need to review something or need an opinion from a TSE, please contact them through slack.

# :rotating_light: :rotating_light: :rotating_light: :construction: Invalid Invalid :construction: :rotating_light: :rotating_light: :rotating_light:

**Comment by chrystalme** [comment]
> # 🚨 🚨 🚨 🚧 Status: Invalid 🚧 🚨 🚨 🚨
> The project is marked as invalid for the next reasons.
> 
> * [ ]  PR should be descriptive.
> * [ ]  Project should have a professional README.
> * [ ]  Project should be a working finished version.
> * [ ]  All linter checks should Pass.
> 
> DON'T submit your project until the project is finished. If you need to review something or need an opinion from a TSE, please contact them through slack.
> 
> # 🚨 🚨 🚨 🚧 Invalid Invalid 🚧 🚨 🚨 🚨

**Comment by chrystalme** [comment]
Hello TSE, I don't know what you mean. My project is descriptive and the read me is from the template

File: index.html [hunk 1]
```diff
@@ -0,0 +1,859 @@
...
+                </div>
+            </div>
+            <div>
+                <nav class="header__md">
+                    <div class="site__nav-left">
+                        <!-- <div class="site__nav-logo">
+                            <a href="#">
+                                <img src="/images/tnw.svg" alt="The New Web">
+                            </a>
+                        </div> -->
```
Comment (mahmoud717) [inline index.html:46]: - the code should not include any commented code. please fix this

File: index.html [hunk 2]
```diff
@@ -0,0 +1,859 @@
...
+                        </div>
+                        <div class="header__md-icon">
+                            <i class="fas fa-search"></i>
+                        </div>
+                    </div>
+                    <div class="site__nav-right">
+                        <div class="site__nav-right-search">
+                            <!-- <a href="#">
+                                <i class="fas fa-search"></i>
+                            </a> -->
```
Comment (mahmoud717) [inline index.html:64]: - the code should not include any commented code. please fix this

File: index.html [hunk 3]
```diff
@@ -0,0 +1,859 @@
...
+                <div class="site__nav-logo">
+                    <a href="#">
+                        <img src="/images/tnw.svg" alt="The New Web">
+                    </a>
+                </div>
+            </div>
+            <div class="site__nav-right">
+
+            </div>
+        </nav> -->
```
Comment (mahmoud717) [inline index.html:86]: - the code should not include any commented code. please fix this

**Review by mahmoud717 — CHANGES_REQUESTED** [review]
### Changes required ♻️ 

**Hello @chrystalme  & @tumwinep**

Great job with the project so far it looks amazing, but there are still some changes to be made:
 
---
### Requested 🔴 

### General

- [ ] the pr should have a descriptive description. 

- [ ] please take into consideration the cloned page will not be approved unless it looks exactly like the original one. that means that it should have the same item positioning, dimensions, responsiveness, appearance, and margins.

- [ ] also, your project should follow the mobile-first approach. meaning that it should flawless in mobile view.

- [ ] the tablet view looks weird
asafsfas

### navbar 

- [ ] the navbar disappears at the mobile view 

### showcase section

- Large version

- [ ] It contains one bigger image on the leftmost part of the row (around two thirds of the width)

- [ ] It contains two smaller images on the rightmost part of the row (one above another) (link to page)

- [ ] It contains TWN logo at the top left corner of the row (link to page)

- medium version 

- [ ] TWN logo disappears

- small version

- [ ] All images take the maximum width and are aligned in one column


###  latest news section


- large version

- [ ] it should contain 4 articles per row




### latest deals section

- [ ] the margins around and between the articles should look exactly like the original one.


---

please make these changes and request another code review.

if you have any questions don't hesitate to contact me on slack **MahmoudMohammad**

**Pull request merged**